_ont_xgs/ont-fs-XGS-ONU-25-20NI.md

```diff
@@ -0,0 +1,586 @@
+
+The UART can be accessed by any of the following methods.
+- Touch the needle to a specific point
+- Shorting a specific pad to access from SFP
+
+### Access from PCB
+The UART can be accessed by connecting a wire or touching a needle to the following points.
+
+{% include image.html file="XGS-ONU-25-20NI/UART_needle.png" alt="XGS-ONU-25-20NI UART Touch point" caption="XGS-ONU-25-20NI UART Touch point" %}
+
+### Access from SFP
+By shorting these two points with solder, you can access the UART from SFP pins 2 and 7.
+
+| USB - TTL Adapter     | SFP Connector (Molex, etc) |
+| --------------------- | -------------------------- |
+| VDD (3.3V)            | pin #15 , #16              |
+| TX                    | pin #7                     |
+| RX                    | pin #2                     |
+| GND                   | pin #10                    |
+
+{% include alert.html content="USB TTL adapter may not work due to insufficient power supply. If possible, obtain 3.3V from the power supply device instead of the USB TTL adapter." alert="Note"  icon="svg-info" color="blue" %}
+
+{% include image.html file="XGS-ONU-25-20NI/UART_SFP.png" alt="XGS-ONU-25-20NI UART Short point" caption="XGS-ONU-25-20NI UART Short point" %}
+
+{% include image.html file="XGS-ONU-25-20NI/UART_bridge.png" alt="XGS-ONU-25-20NI UART solder bridge" caption="XGS-ONU-25-20NI UART solder bridge" %}
+
+
+## Firmware is interchangeable with
+The firmware is compatible with the following.
+- CIG XG-99S
+- CIG XE-99S (10G-EPON mode)
+- ECIN EN-XGSFPP-OMAC v1
+
+
+## List of software versions
+Available from `#ONT/system>ontver build` and `#ONT/system/shell>cat /etc/version`.
+- R4.4.20.016 (from FS.com docs)
+- R4.4.20.018 - 20230426061821
+- R4.4.20.022 - 20230612033149
+
+
+## List of partitions
+
+| dev    | size     | erasesize | name              |
+| ------ | -------- | --------- | ----------------- |
+| mtd0   | 00040000 | 00020000  | "ssb"             |
+| mtd1   | 00002000 | 00020000  | "uboot-env"       |
+| mtd2   | 00002000 | 00020000  | "dtb0"            |
+| mtd3   | 0003c000 | 00020000  | "kernel0"         |
+| mtd4   | 00300000 | 00020000  | "rootfs0"         |
+| mtd5   | 004c0000 | 00020000  | "dtb1"            |
+| mtd6   | 00300000 | 00020000  | "kernel1"         |
+| mtd7   | 004c0000 | 00020000  | "rootfs1"         |
+| mtd8   | 00001000 | 00020000  | "userdata"        |
+| mtd9   | 00001000 | 00020000  | "mfginfo1"        |
+| mtd10  | 00001000 | 00020000  | "mfginfo2"        |
+| mtd11  | 00001000 | 00020000  | "uboot-env2"      |
+
+This ONT supports dual boot. 
+
+`kernel0` and `rootfs0` respectively contain the kernel and firmware of the first image, `kernel1` and `rootfs1` the kernel and the firmware of the second one.
+
+
+# Usage
+
+## Login and Enable
+{% include alert.html content="This is an external file ([emulate ONT in QEMU](https://github.com/YuukiJapanTech/CA8271x)), so use it at your own risk!" alert="Note"  icon="svg-info" color="blue" %}
```

Comment:
```
```suggestion
## Login and Enable

You can use the following form to generate login credentials

{% include cig_password_xgspon.html password_len="8" %}

Or following this externa file that [emulate ONT in QEMU](https://github.com/YuukiJapanTech/CA8271x), so use it at your own risk.